cmd/kubeadm/app/master/manifests_test.go

```diff
@@ -206,7 +206,7 @@ func TestK8sVolume(t *testing.T) {
 				Name: "k8s",
 				VolumeSource: api.VolumeSource{
 					HostPath: &api.HostPathVolumeSource{
-						Path: kubeadmapi.GlobalEnvParams.KubernetesDir},
+						Path: kubeadmconstants.KubernetesDir},
```

Comment:
This will result in a wrong behavior. The path used earlier was `${TMPDIR}/${CONSTANT_PATH}` which now is replaced with `${CONSTANT_PATH_ONLY}`

You should make a variable `kubernetesDir := filepath.Join(tmpdir, kubeadmconstants.KubernetesDir)` for all testing in this PR